onebone/signal/filter.py

```diff
@@ -275,6 +275,94 @@ def bandpass_filter(
     return signal
 
 
+def bandpass_filter_ideal(
+    signal: np.ndarray,
+    fs: Union[int, float],
+    l_cutoff: Union[int, float],
+    h_cutoff: Union[int, float],
+) -> np.ndarray:
+    """
+    .. note:: This method may cause distortion of signal. Generally, this operates well on signals extracted in low resolution. \
```

Comment:
```suggestion
    .. warning:: This method **may cause distortion of signal**. \
        Generally, this operates well on signals extracted in low resolution. \
        In order to check the distortion of signals, \
        it is recommended to monitor the linear transition of phase.
```
note 보다는 warning으로 변경하면 좋겠습니다. 위 코드처럼 변경하면 아래 그림처럼 문서가 나옵니다.
![image](https://user-images.githubusercontent.com/86699249/177486328-76beff68-5e6c-4d31-a2f5-4a27060a8ffe.png)

Reply:
@ybpark-onepredict 이부분 마지막으로 확인 부탁드립니다.